renderdoc/os/posix/linux/linux_stringio.cpp

```diff
@@ -22,6 +22,7 @@
  * THE SOFTWARE.
  ******************************************************************************/
 
+#include <cctype>
```

Comment:
I didn't comment on each include to avoid spam, but this should similarly be `<ctype.h>`.

Reply:
Done